### ISSUE TITLE
Regression Model

### ISSUE THREAD

**Issue by ShivamSharma1997** [ThilinaRajapakse/simpletransformers#704]
Hi,
I was working with the regression model and was wondering if I need to apply softmax function to the predictions/raw outputs to get a result between 0 and 1 as all my training targets are between 0 and 1 but the resulting predictions have values greater than 1.

**Comment by ThilinaRajapakse** [comment]
No, you shouldn't need to apply a softmax to the predictions. How much training data do you have?

**Comment by ShivamSharma1997** [comment]
@ThilinaRajapakse Over a 100k of text.

**Comment by ThilinaRajapakse** [comment]
Can you provide a minimal snipped of your training code, please?
